autogpt_platform/frontend/src/components/integrations/credentials-input.tsx

```diff
@@ -56,6 +58,7 @@ export const providerIcons: Record<
   ideogram: fallbackIcon,
   llm: fallbackIcon,
```

Comment:
we should be able to remove this once we get this right

Reply:
Now `llm` is removed